src/routes/plans/[id]/+page.svelte

```diff
@@ -242,6 +257,18 @@
     clearSchedulingErrors();
   }
 
+  function onCloseSnapshotPreview() {
+    $planSnapshotId = null;
```

Comment:
I think your approach of not loading any sim dataset after closing the preview makes sense but just wanted to double check that we shouldn't be loading the latest matching sim dataset if found?

Reply:
Yeah. That's a great point. It probably should autoselect the latest